tyr/servers/mongo/data.py

```diff
@@ -77,18 +77,26 @@ def bake(self):
                 {
                     'user': 'mongod',
                     'group': 'mongod',
-                    'size': 10,
-                    'iops': 0,
+                    'size': self.data_volume_size,
+                    'iops': self.data_volume_iops,
+                    'device': '/dev/xvdf',
+                    'mount': '/volr'
+                },
+                {
+                    'user': 'mongod',
+                    'group': 'mongod',
+                    'size': 25,
+                    'iops': 250,
```

Comment:
Also, would it make sense to have arguments and variables `journal_volume_iops` and `journal_volume_size`?